test-harness/gold-master-files/__path____no_auth_pets_123___method___POST___headers____Content-Type___application_x-www-form-urlencoded____body___name=just_a_name__.json

```diff
@@ -9,24 +9,24 @@
     "host": "http://localhost:4010"
   },
   "response": {
-    "status": 415,
-    "statusText": "Unsupported Media Type",
+    "status": 500,
```

Comment:
Is this meant to be 500 or 415?

Reply:
Fixed in 399d276bcca3a5fcdd0c7bcfaff47d339d96f72c. This is yet another problem of the test harness, it does not have any automatic invalidation system and so they can become stale and we do not notice it. We really need to find another way here.

Reply:
I gotcha. Lets pair up sometime on Monday to chat about what we think we'd like to do to improve test harness.